R/deps.R

```diff
@@ -109,7 +109,7 @@ dev_package_deps <- function(pkg = ".", dependencies = NA,
   deps <- unlist(lapply(parsed, `[[`, "name"), use.names = FALSE)
 
   if (is_bioconductor(pkg)) {
-    check_suggested("BiocInstaller")
+    check_bioconductor()
```

Comment:
They are likely packages that were rejected for Bioconductor for whatever reason and the authors did not remove the `BiocViews` field.

We could add a bioconductor argument to `dev_package_deps()` which would allow you to turn off the check like we have in `check_cran()`.